server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -162,6 +164,7 @@
   private final TabletServer server;
   private final WriteTracker writeTracker = new WriteTracker();
   private final RowLocks rowLocks = new RowLocks();
+  private final ServerBulkImportStatus bulkImportStatus = new ServerBulkImportStatus();
```

Comment:
I'm not sure I understand the point of this addition here. I see the three places where you update/remove entries being tracked in this object, but I don't see anywhere where the statuses being tracked in this ever get read and used.

Reply:
This was an unfinished attempt to mirror what was done in `ClientServiceHandler`:
https://github.com/apache/accumulo/blob/09eef7b32da3d1c70709483c5c5a1645e3c64a44/server/base/src/main/java/org/apache/accumulo/server/client/ClientServiceHandler.java#L78

Turns out it was even easier for `ThriftClientHandler`. Fixed in d3c65cf